docs/topics/forms/formsets.txt

```diff
@@ -509,6 +509,16 @@ On the other hand, if you are using a plain ``FormSet``, it's up to you to
 handle ``formset.deleted_forms``, perhaps in your formset's ``save()`` method,
 as there's no general notion of what it means to delete a form.
 
+``can_delete_extra``
+~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
missing `.. versionadded:: 1.9` and a mention in the release notes

(Please review your patch using the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).)